src/app/components/tabs.tsx

```diff
@@ -1,38 +1,39 @@
-import { Box as RBox, Tabs as RTabs } from '@radix-ui/themes';
-import { styled } from 'leather-styles/jsx';
+import { Box } from 'leather-styles/jsx';
 
 import { capitalize } from '@app/common/utils';
+import { Tabs as LeatherTabs } from '@app/ui/components/tabs/tabs';
 
-interface Tabs {
+interface RouteTabs {
```

Comment:
I was confused by the name and placement of this at first. Maybe we can move it to `ui/components/tabs` ? 

As per discussion [here](https://github.com/leather-wallet/extension/pull/4789#discussion_r1446622670) this would be a valid case of having another story under `Tabs` for `RouteTabs` 

Then under the `Tabs` section we would have
- `Tabs` (generic tabs)
- `RoutableTabs` (routable tabs)

Reply:
Yeah wasn't sure what to do here and wonder if we should remove this entirely. This tabs component does a few things extra that the radix component doesn't:
- Accepts an `activeTab` prop
- Has an onClick handler
- Sets a `data-testid`

I'd probably err on the side of this completely, and composing the tabs directly from the styled tabs in each case.

Reply:
@pete-watters @fbwoolf see my second commit. Hope you agree this is a big improvement. IMO this is the way we should try and use the radix primitives. The old tabs component created some wrapper around it which makes it clunky and less composable. Here, I've removed it and I compose the tabs the way Radix intended. This means the API of our tabs is identical to Radix.

Our tabs are now Radix tabs + Leather styles and nothing more.

Reply:
This is a great improvement, thanks for re-working it. It's much cleaner and a good pattern for us to use for our other components 🙇